packages/v2/backend/src/k9sak/kodeverk/UttakUtfall.ts

```diff
@@ -0,0 +1,12 @@
+import type { Kodeverk } from '../../shared/Kodeverk.ts';
+import { type UttakDto } from '../generated';
+
+export type UttakUtfallType = UttakDto['utfall'];
+
+export type UttakUtfallTypeKodeverk = Kodeverk<UttakUtfallType, 'UTTAK_UTFALL'>;
+
+export const uttakUtfallType: Readonly<Record<UttakUtfallType, UttakUtfallType>> = {
```

Comment:
Når ein generert enum type string union har like prop namn og verdier, slik som i dette tilfellet, så kan ein og finne den korrekte konstanten i den generte kildekoden, og bruke den istadenfor å kode den manuelt. Slik som dette:
```typescript
import { type UttakDto, utfall2 } from '../generated';
export type UttakUtfallType = UttakDto['utfall'];
export const uttakUtfallType: Readonly<Record<UttakUtfallType, UttakUtfallType>> = utfall2
``` 
Då vil vi unngå evt framtidige problem viss det blir lagt til verdi i enum.

Det ein må passe på er å importere korrekt konstant, pr no blir det desverre generert fleire med namnekollisjon, så i dette tilfellet ser det ut til å vere `utfall2` som er korrekt.

Du kan vurdere om du vil endre til dette her eller ikkje, men ville informere om muligheten i alle fall.

Reply:
Merk at du må merge in master på branchen før dette er mulig.